### PR TITLE
Additional fixes

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2015-11-11 at 16 20 19|screen shot 2015-11-11 at 14 47 21|
|screen shot 2015-11-11 at 16 20 22|screen shot 2015-11-11 at 14 50 33|
||screen shot 2015-11-11 at 16 21 23|
|screen shot 2015-11-11 at 16 35 59|screen shot 2015-11-11 at 16 39 52|

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+1.0.6 (Unreleased)
+------------------
+
+* Fix hover and selected lang link color
```
Comment (vxsx) [inline CHANGELOG.rst]: in some specific app?

**Comment by vxsx** [comment]
LGTM

File: CHANGELOG.rst (hunk 1 above)
Comment (lory87) [inline CHANGELOG.rst]: @vxsx updated the comment in changelog. thank you

**Comment by vxsx** [comment]
:+1:

**Pull request merged**